src/puppetlabs/puppetdb/command.clj

```diff
@@ -97,8 +97,8 @@
 
 (def supported-command-versions
   {"replace facts" (version-range 2 4)
-   "replace catalog" (version-range 4 7)
-   "store report" (version-range 3 6)
+   "replace catalog" (version-range 4 8)
```

Comment:
Hmm, have we lost any form of deprecation warnings for old commands now? I can see warn-deprecated, but its not used. Anyway, I think we don't drop deprecation messages anymore for older stuff ...